CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ release.
 ## [Unreleased]
 
 ### Added
-
+- The following calibration applications were updated to not require local mission-specific SPICE kernels when working with spiceinited cubes: amicacal, ctxcal, lrowaccal, moccal, mdiscal, hical, hicalbeta, vikcal, and gllssical. This makes it possible to first run spiceinit using the spice server and then run these calibration applications without ever needing to download mission-specific kernels. [#4303](https://github.com/USGS-Astrogeology/ISIS3/issues/4303)
```

Comment:
Probably should talk about this being a new feature, but the old behavior is still there. If spiceinit has not been run then the apps run as they used to and kernels area is still required

Reply:
Good feedback, thanks! Updating...